assignment02/Exercise1.java

```diff
@@ -0,0 +1,31 @@
+package assignment02;
+
+import java.util.Arrays;
+
+public class Exercise1 {
+
+    public static int getNumber(int[] nums, String value) {
+        int result = 0;
+        Arrays.sort(nums);   
+        switch (value) {
+            case "min":
+                result = nums[0]; 
+                break;
+            case "max":
+                result = nums[nums.length - 1];
+                break;
+            default:
+                return (int) ((nums[0] + nums[nums.length - 1]) / 2);
```

Comment:
Until here the solution is very awesome! and smart. But :worried:  I think the average is to sum all numbers divided by total quantity of numbers

Reply:
jajaja my mistake =P